fades/main.py

```diff
@@ -260,6 +260,9 @@ def go(argv):
     # run forest run!!
     python_exe = 'ipython' if args.ipython else 'python'
     python_exe = os.path.join(venv_data['env_bin_path'], python_exe)
+    # add the virtualenv /bin path to the child PATH.
+    os.environ['PATH'] = "{}:{}".format(venv_data['env_bin_path'], os.environ.get('PATH'))
+    logger.debug("child PATH: %s", os.environ)
```

Comment:
we are using `logger` everywhere, so I think is better to change the imported module nanme